windows-service-guide.md

```diff
@@ -0,0 +1,85 @@
+# AlloyDB Auth Proxy Windows Service Guide
+
+This document covers running the *AlloyDB Auth Proxy* as service
+on the Windows operating system.
+
+It was originally built and tested using Go 1.20.2 on Windows Server 2019.
+
+## Install the Windows Service
+
+Prerequisites: A built binary for Windows of the AlloyDB Auth Proxy is required. Either build it from source or [download a release](https://github.com/GoogleCloudPlatform/alloydb-auth-proxy/releases) of a Windows pre-built version, e.g. `alloydb-auth-proxy.x64.exe`.
```

Comment:
nit: could we wrap these at 80-ish characters? It helps people read this in the terminal.